data/relation-godollo.yaml

```diff
@@ -30,6 +41,17 @@ filters:
     # 101: sima 101 nincs, csak 101/a, 101/b, 101/c.
     # 105: sima 105 nincs, csak 105/a, 105/b, 105/c.
     invalid: ['15', '19', '25b', '29a', '29b', '29c', '29d', '29f', '99', '101', '103', '105']
+  Szent János utca:
+    # 1: sima 1 nincs, csak 1/a, 1/b, 1/c, 1/d.
+    # 2: sima 2 nincs, csak 2/a, 2/b.
+    # 4: sima 4 nincs, csak 4/a, 4/b.
+    # 6-8: 6-8 nincs, csak 6, 8/a, 8/b
+    # 8: sima 8 nincs, csak 8/a, 8/b.
+    # 11: sima 11 nincs, csak 11/a, 11/b, 11/c.
+    # 12: sima 12 nincs, csak 12/a, 12/b.
+    # 14: sima 14 nincs, csak 14/a, 14/b.
+    # 16: sima 16 nincs, csak 16/a, 16/b.
+    invalid: ['1', '2', '4', '6-8', '8', '11', '12', '14', '16']
```

Comment:
U.ez a baj itt is, a 6-8 nem jó egy invalid elemként, elég csak a 6.

Bővebb infó:

- https://wiki.openstreetmap.org/wiki/Hungary/osm-gimmisn
- https://vmiklos.hu/osm-gimmisn/usage.html#filtering-out-incorrect-information

Reply:
Holnap meglesz, hacsak addig @vasony nem előz meg :)

Reply:
csak hogy igazad legyen :)